Volume/View Models/ArticleInfo.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  ArticleInfo.swift
+//  Volume
+//
+//  Created by Cameron Russell on 10/30/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+struct ArticleInfo: View {
+    let article: Article
+    private let lineLimit = 3
```

Comment:
I think we should put private variables on top. So private vars, then public vars, alphabetized within each category. Also newline after struct def.